src/web/lib/hakuneko/frontend@classic-dark/chapters.html

```diff
@@ -94,12 +94,12 @@
             <table border="0" cellpadding="0" cellspacing="0" width="100%">
```

Comment:
No need to change this file at all when doing
https://github.com/manga-download/hakuneko/pull/448/files#r329122054